src/DacpacTool/Program.cs

```diff
@@ -138,6 +139,23 @@ private static int BuildDacpac(BuildOptions options)
                 packageBuilder.AddWarningsToSuppress(options.SuppressWarnings);
             }
 
+            // Add warnings suppressions for particular files through $Project.WarningsSuppression.txt
+            if (options.SuppressWarningsListFile != null)
+            {
+                if (options.SuppressWarningsListFile.Exists)
```

Comment:
Perhaps this logic should move into the `PackageBuilder` as well. `Program` is merely responsible for parsing command line arguments and passing them to the `PackageBuilder `.

Reply:
I make this by analogy with how it is processed standard `InputFiles.txt`. `Program` parse this file and pass to `PackageBuilder` particular paths.

Reply:
Funny how patterns like this develop from a series of small incremental steps... I based the parsing of `InputFiles.txt` off of the code that existed at the time, which accepted multiple `--input` arguments and sent each to `PackageBuilder.AddInputFile()`. Now that we have multiple of these types of input files, though, I tend to agree that this belongs in `PackageBuilder`. I think the way this was developed makes sense according to the patterns in our code right now, but maybe we should refactor in a follow-up to this PR?